src/material/tabs/tab-group.ts

```diff
@@ -84,7 +84,8 @@ const _MatTabGroupMixinBase: CanColorCtor & CanDisableRippleCtor & typeof MatTab
   templateUrl: 'tab-group.html',
   styleUrls: ['tab-group.css'],
   encapsulation: ViewEncapsulation.None,
-  changeDetection: ChangeDetectionStrategy.OnPush,
+  // tslint:disable-next-line:validate-decorators
```

Comment:
Should we have a comment about why we're disabling it? Also do we need to do this on all components or just the one that's around the template content?

Finally, the setup between tabs and stepper is pretty similar so my guess is that we might have to do the same there.

Reply:
Yeah, I suspect we'll be doing this for any component that consumes a template, in which case we would just disable the lint. I wanted to do that in a separate PR.